analyzer/src/funTest/assets/projects/external/directories-expected-output.yml

```diff
@@ -31,6 +31,8 @@ analyzer:
       - "Mozilla Public License 2.0"
       declared_licenses_processed:
         spdx_expression: "MPL-2.0"
+        mapped:
```

Comment:
What do you guys think about making this property  `mapping` instead of `mapped` and make it contain both, unmapped and mapped entries (excluding identity mapping entries), for simpler parsing.

Reply:
Then `declared_licenses_mapping` and `declared_licenses_processed` would be on the same level, right? I believe that makes it less clear that `declared_licenses_mapping` belongs to `declared_licenses_processed`, so I prefer the current approach.

Reply:
> Then declared_licenses_mapping and declared_licenses_processed would be on the same level, right?

Yes.

Reply:
I agree with @sschuberth here, I prefer to keep the mappings inside `declared_licenses_processed`.
For the future we could think about removing `spdx_expression` and instead making the `mapping` complete, because the expression could always be generated from that. But right now I only wanted to fill the whole that we lose information about the mappings.